src/main/java/io/papermc/codebook/lvt/LvtAssignmentSuggester.java

```diff
@@ -478,4 +480,35 @@ private static boolean isStringAllUppercase(final String input) {
 
         return null;
     }
+
+    // 3 instructions, load "this" local var, getfield, return - TODO maybe if there is a CAST,
+    private static final IntPredicate[] OPCODES_IN_ORDER = new IntPredicate[] {
+        i -> i == Opcodes.ALOAD, i -> i == Opcodes.GETFIELD, i -> i >= Opcodes.IRETURN && i <= Opcodes.RETURN
+    };
+
+    private static @Nullable String suggestNameFromFluentGetter(
+            final AsmClassData owner, final AsmMethodData method, final MethodInsnNode insn) {
+        // I think it's best to only work with primitive types here, as other types should already have names
+        // and this dramatically cuts down on the number of methods analyzed because we aren't filtering by
+        // method name
+        if (!(method.descriptor().getReturnType() instanceof PrimitiveType)) {
```

Comment:
I'd also not apply this to static methods.

Reply:
Yeah, I logged all the static methods it was applying to, and only 1 was important, `System#currentTimeMillis` so I just singled that one out. Also I added a check to make sure the method didn't have any parameters.